doc/source/cluster/kubernetes/examples/rayjob-kueue-gang-scheduling.md

```diff
@@ -37,16 +37,16 @@ Create a GKE cluster with the `enable-autoscaling` option:
 ```bash
 gcloud container clusters create kuberay-gpu-cluster \
     --num-nodes=1 --min-nodes 0 --max-nodes 1 --enable-autoscaling \
-    --zone=us-west1-b --machine-type e2-standard-4 --cluster-version 1.29
+    --zone=us-east4-c --machine-type e2-standard-4
```

Comment:
I switched the zone just based on personal experience, this zone has less stockouts  for L4 GPUs